src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -431,9 +429,9 @@ public void MultiNodeSpecAfterAll()
             if (SelfIndex == 0)
             {
                 TestConductor.RemoveNode(_myself);
```

Comment:
This method is now called on shutdown via `Dispose`, and orders the shutdown of the nodes such that the controller is always last to exit.